assets/css/styles.css

```diff
@@ -0,0 +1,322 @@
+}
+
+/* Navbar */
+.navbar {
+    width: 30rem;
+}
+
+/* Style for navbar links and contact me list */
+ul {
+    display: flex;
+    list-style-type: none;
+    justify-content: space-between;
+    padding-inline-start: 0;
+}
+
+/* Style for banner image */
+.banner {
+    display: flex;
+    width: 100%;
+    height: 30rem;
+    background-image: url('../images/banner.jpg');
+    color: var(--light);
+    background-size: cover;
+    background-position: center;
+    align-items: center;
+}
+
+/* Style for title on the banner image */
+.banner h2 {
+    font-size: 3rem;
+    width: 100%;
+    text-align: center;
+    text-shadow: 0rem 0rem 0.5rem var(--black);
+}
+
+/* Style for each content section */
+.content {
+    display: flex;
+    margin: 4rem 7rem;
+}
+
+/* Style for title of the content sections */
+.content-title {
+    font-size: 2rem;
+    width: 15%;
+    text-align: right;
+    margin-right: 1rem;
+}
+
+/* Style for content box */
+.content-box {
+    display: flex;
+    width: 85%;
+    padding-left: 2rem;
+    border-left: 0.3rem solid var(--red);
+    align-items: center;
+}
+
+/* Style for About me image, targeting using ID */
+#about-me img {
+    border-radius: 50%;
+}
+
+/* Style for paragraph in about me section */
+#about-me p {
+    margin-left: 2rem;
+    font-size: 1.4rem;
+    line-height: 2.2rem;
+}
+
+/* Style for flexbox in My work */
+#my-work .content-box {
+    flex-grow: 1;
+    flex-wrap: wrap;
+    justify-content: space-between;
+}
+
+/* Style for first box in My Work section */
+.first-project {
+    height: 30rem;
+    width: 100%;
+    margin-bottom: 3rem;
+}
+
+.other-project {
+    height: 20rem;
+    width: 20rem;
+}
+
+/* To fit and position the image properly */
+.first-project img, .other-project img {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+    object-position: 0% 0%;
+}
+
+/* Style for project information */
+.project-info {
+    background-color: var(--darkGrey);
+    color: var(--light);
+    padding: 0.8rem;
+    position: absolute;
+    bottom: 10%;
+}
+
+h4 {
+    font-size: 2rem;
+}
+
+.project-info p {
+    font-size: 1.3rem;
+}
+
+/* Style for all projects */
+.first-project, .other-project {
+    position: relative;
+    border-style: solid;
+    border-color: var(--darkGrey);
+    border-width: 0.5rem;
+    filter: brightness(50%);
+}
+
+/* Shadow and brighten up for project boxes */
+.first-project:hover, .other-project:hover {
+    filter: brightness(100%);
+    box-shadow: 0rem 0rem 0.8rem var(--red);
+}
+
+/* Style for Contact me section */
+#contact-me ul {
+    width: 100%;
+    justify-content: space-evenly;
+    font-size: 1.5rem;
+}
+
+/* To remove blue default color on links */
+#contact-me a {
+    color: var(--black)
+}
+
+/* Duration of transitions */
+#contact-me a, #about-me p, .first-project, .other-project {
+    transition-duration: 0.2s;
+}
+
+#contact-me a:hover, #about-me p:hover {
+    text-shadow: 0rem 0rem 0.7rem var(--red);
+}
```

Comment:
Consider taking the hover of the `About Me` text. As per the feedback, consider only using hovers when elements do something e.g. a link or button.

Reply:
Yeah that's understandable, thanks for feedback! 👍